app/views/admin/navigation_links/_add_navigation_link_modal.html.erb

```diff
@@ -0,0 +1,54 @@
+<button class="btn btn-primary" type="button" onclick="toggleNavigationLinkModal()">
+  Add Navigation Link
+</button>
+
+<div id="add-navigation-link-modal" class="hidden">
+  <div class="crayons-modal crayons-modal--s">
+    <div class="crayons-modal__box">
+      <header class="crayons-modal__box__header">
+        <h2>Add Navigation Link</h2>
+        <button type="button" onclick="toggleNavigationLinkModal()" class="crayons-btn crayons-btn--icon crayons-btn--ghost">
+          <svg width="24" height="24" viewBox="0 0 24 24" class="crayons-icon" xmlns="http://www.w3.org/2000/svg">
+            <path d="M12 10.586l4.95-4.95 1.414 1.414-4.95 4.95 4.95 4.95-1.414 1.414-4.95-4.95-4.95 4.95-1.414-1.414 4.95-4.95-4.95-4.95L7.05 5.636l4.95 4.95z" />
+          </svg>
+        </button>
+      </header>
+      <div class="crayons-modal__box__body">
+        <div class="form-group grid p-6 mb-6 gap-1">
+          <div class="form-group grid p-6 mb-6 gap-1">
+            <%= form_for(NavigationLink.new, url: admin_navigation_links_path) do |form| %>
+              <div class="form-group">
+                <%= form.label :name %>
+                <%= form.text_field :name, class: "form-control" %>
+              </div>
+              <div class="form-group">
+                <%= form.label :url %>
+                <%= form.text_field :url, class: "form-control" %>
+              </div>
+              <div class="form-group">
+                <%= form.label :icon %>
+                <%= form.text_field :icon, class: "form-control" %>
```

Comment:
I think we should say that the icon should be a SVG in the explanation, as I obviously used a png and had to start from scratch when presented the error :( 
This should also be a text area I guess...

Reply:
Definitely, in the PR I'm working on currently, I've added an explanation for the page and for the fields. I wanted to get the foundation working in this PR. And thanks, changed to text_area ✅